tests/Integration/User/Application/EventSubscriber/UserRegisteredEventSubscriberTest.php

```diff
@@ -29,7 +29,7 @@ protected function setUp(): void
     public function testConfirmationEmailSent(): void
     {
         $userId = $this->faker->uuid();
-        $emailAddress = $this->faker->email();
+        $emailAddress = 'test@example.com';
```

Comment:
We should use faker instead of hardcoding the test data
It will be better to return back this line